policyengine_core/tools/test_runner.py

```diff
@@ -158,7 +158,7 @@ class YamlItem(pytest.Item):
     def __init__(
         self, name, parent, baseline_tax_benefit_system, test, options
     ):
-        super(YamlItem, self).__init__(name, parent)
+        super(YamlItem, self).from_parent(parent, name=name)
```

Comment:
`from_parent` is a classmethod that returns an `Item`/`Node` instance, so that's why this isn't working/correct. I haven't yet become familiar with the test-suite architecture, but seems like a little tweaking is needed to separate the item/node construction from initialization (or could do some clever, yet potentially confusing, usage of `__new__()`